services/site/src/pages/legal-notice.tsx

```diff
@@ -5,19 +5,19 @@ import Head from "next/head";
 import Link from "next/link";
 import React from "react";
 
-const Imprint: React.FunctionComponent = () => {
+const LegalNotice: React.FunctionComponent = () => {
   return (
     <>
       <Head>
-        <title>Imprint | a11yphant</title>
+        <title>Legal Notice | a11yphant</title>
         <meta name="robots" content="noindex,nofollow" />
-        <meta name="description" content="a11yphant is a master project by 6 students at the University of Applied Sciences Salzburg." />
+        <meta name="description" content="a11yphant is an interactive online course for web accessibility." />
```

Comment:
I don't think that the About page and the legal notice page should have the same SEO text. I mean we have a no index no follow here, so it might doesn't even matter, just not entirely sure about it. I thought the SEO text has to be different for every page.

Reply:
The SEO description doesn't matter anyway. It's only the text shown under the search result and since we have a noindex we could as well remove it completely

https://www.tributemedia.com/blog/meta-description-faqs#toc_2